include/vctr/Containers/Array.h

```diff
@@ -89,7 +89,7 @@ class Array : public VctrBase<ElementType, std::array<ElementType, storageExtent
     {}
 
     /** Creates an Array with extent 1 from an initial value. */
-    template <is::suitableSingleElementInitializer T>
+    template <std::convertible_to<ElementType> T>
```

Comment:
Assuming this also fixes the assignment of `std::array` to `vctr::Array` (which was my original issue)? 🙂

Reply:
awesome!